WordPress/Classes/ViewRelated/Menus/Views/MenuItemSourceCell.m

```diff
@@ -0,0 +1,147 @@
+#import "MenuItemSourceCell.h"
+#import "WPStyleGuide.h"
+#import "WPFontManager.h"
+#import "Menu+ViewDesign.h"
+
+#pragma mark - MenuItemSourceRadioButton
+
+@interface MenuItemSourceRadioButton : UIView
+
+@property (nonatomic, assign) BOOL selected;
+@property (nonatomic, assign) BOOL drawsHighlighted;
+
+@end
+
+#pragma mark - MenuItemSourceOptionView
+
+static CGFloat const MenuItemSourceCellHierarchyIdentationWidth = 17.0;
+
+@interface MenuItemSourceCell ()
+
+@property (nonatomic, strong, readonly) UIStackView *stackView;
+@property (nonatomic, strong, readonly) UIStackView *labelsStackView;
+@property (nonatomic, strong, readonly) UILabel *label;
+@property (nonatomic, strong, readonly) NSLayoutConstraint *leadingLayoutConstraintForContentViewIndentation;
+@property (nonatomic, strong, readonly) NSLayoutConstraint *topLayoutConstraintForContentViewIndentation;
+@property (nonatomic, strong, readonly) NSLayoutConstraint *topLayoutDefaultConstraint;
+
+@end
+
+@implementation MenuItemSourceCell
+
+- (void)dealloc
+{
+    [[NSNotificationCenter defaultCenter] removeObserver:self];
+}
+
+- (id)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
```

Comment:
This is the `UITableViewCell` initializer, but updating to use `instancetype` instead of `id`.